samples/apps/auth-api-webapp-react/README.md

```diff
@@ -9,19 +9,19 @@
 
 ## Running the sample
 
-1. You will need an [Open AI Key](https://openai.com/api/) or
+1. You will need an [Open AI Key](https://platform.openai.com/account/api-keys) or
```

Comment:
This link takes you to a "Authentication required" page, whereas the previous one is just the public api landing page. I see why you'd want to navigate people directly to the keys page, but if they're not already signed up (and many won't be), it's probably not the ideal experience.

Reply:
@shawncal I changed that because today when you get into the /api you are redirected to /product and there is not clear where this APIs keys are making a potential 1 step in a 4 step process.

/api (4 step process)

1. Click in `for developers` that will bring the page to: https://openai.com/product#made-for-developers

2. Click in `Get Started` 

3. They are presented to an Authentication required page (little similar to the proposed experience).
![image](https://user-images.githubusercontent.com/19890735/224259112-9e81eb4e-c00d-4fd3-a289-749deb2687e7.png)

4. Need to figure out that the API keys are in the context menu of the user icon on top menu bar.

I have two options here:
1. Bring the link back to the product page, requiring some extra steps and more time / cleverness for non pro-developers to find where is this API Key place.
2. Add an extra description in the readme informing that a Sign Up/Login is required and will be presented before clicking in the API key link.